experimental/c/chechen_latin/source/help/chechen_latin.php

```diff
@@ -0,0 +1,154 @@
+<?php
+$pagename  = 'Chechen Language Keyboard Help';
+$pagetitle = $pagename;
+// Header
+require_once('header.php');
+?>
+
+  <h1>Start Using Chechen Latin Keyboard</h1>
+
+  <p>
+    Chechen Latin Keyboard v1.0.
+  </p>
+  <p>
+
+  </p>
+
+  <h1>Keyboard Layout</h1>
+  <div id="layout"></div>
+
+
+  <script>
+    document.addEventListener('DOMContentLoaded', documentReady());
+
+    function getOS() {
+      let userAgent = window.navigator.userAgent,
+        platform = window.navigator.platform,
+        macosPlatforms = ['Macintosh', 'MacIntel', 'MacPPC', 'Mac68K'],
+        windowsPlatforms = ['Win32', 'Win64', 'Windows', 'WinCE'],
+        iosPlatforms = ['iPhone', 'iPad', 'iPod'],
+        os = null;
+
+      if (macosPlatforms.indexOf(platform) !== -1) {
+        os = 'macos';
+      } else if (iosPlatforms.indexOf(platform) !== -1) {
+        os = 'iphone';
+      } else if (windowsPlatforms.indexOf(platform) !== -1) {
+        os = 'windows';
+      } else if (/Android/.test(userAgent)) {
+        os = 'android';
+      } else if (!os && /Linux/.test(platform)) {
+        os = 'linux';
+      }
+
+      return os;
+    }
+
+    function documentReady() {
+      const layout = document.getElementById('layout');
+
+      const prefix = getOS();
+
+      const mobileLayouts = [{
+        name: 'Lower case layout',
+        layout: '_default',
+        helpText: ''
```

Comment:
@gushmazuko  I've included placeholders here to insert help text for each layout. What do you think, do we have something that is not self-explanatory?